tests_integ/helpers/helpers.py

```diff
@@ -0,0 +1,148 @@
+import json
+import logging
+import re
+import random
+import string  # pylint: disable=deprecated-module
+from functools import reduce
+
+import boto3
+from botocore.exceptions import ClientError, NoRegionError
+
+from samtranslator.model.exceptions import InvalidDocumentException
+from samtranslator.translator.managed_policy_translator import ManagedPolicyLoader
+from samtranslator.translator.transform import transform
+from samtranslator.yaml_helper import yaml_parse
+
+# Length of the suffix sometimes added by CFN to logical IDs
+LOGICAL_RES_SUFFIX_LENGTH = 10
```

Comment:
Is this length documented?

Reply:
After a little digging, this is actually us (SAM) that add this suffix to the Logical IDs when translating.

It's located in `logical_id_generator.py`, I changed the code to remove this constant and use `LogicalIdGenerator.HASH_LENGTH` instead.